### PR TITLE
Add last_edited to v1 objects

### PR DESCRIPTION
**Pull request by skairunner**

### CODE REVIEW

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 1]
```diff
@@ -269,9 +275,10 @@ message TextHeader {
     string file_id = 1; // DEPRECATED: This field will always be empty
     string header = 2; // The text content of the header
     string raw_header = 3; // The raw bytes of a header as a string
+    google.protobuf.Timestamp last_edited = 14;  // The last time this object was created or edited.
```
Comment (audunska) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:278]: Hmm, it's worth thinking about whether timestamps belong in the headers. Semantically, the headers are part of other objects like seismic stores and seismics. If we have headers in both seismic objects and their headers, what does it mean if the timestamps are different?

Comment (skairunner) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:278]: Headers can be edited to amend their data, so I think it makes sense that they could have their own timestamp 🤔

Comment (audunska) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:278]: But editing headers always goes through the edit endpoint of the containing object. The way to edit a seismic object's header is to call `EditSeismic`. I would expect an `EditSeismic` call to change the timestamp of the seismic object. So to me, semantically, the header is part of the seismic object.

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 2]
```diff
@@ -271,7 +277,7 @@ message TextHeader {
     string raw_header = 3; // The raw bytes of a header as a string
 }
 /**
-A representation of binary headers used to create or edit existing headers. BinaryHeader FIELDS contains the list of valid fields. to set after the object is constructed.
+A representation of binary headers used to create or edit exiting headers. BinaryHeader FIELDS contains the list of valid fields. to set after the object is constructed.
```
Comment (audunska) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:280]:
```suggestion
A representation of binary headers used to create or edit existing headers. BinaryHeader FIELDS contains the list of valid fields. to set after the object is constructed.
```

**Review by audunska — APPROVED** [review]

**Pull request merged**